generated/nidaqmx/_task_modules/channels/ai_channel.py

```diff
@@ -4980,8 +4981,7 @@ def ai_gain(self):
     @property
     def ai_impedance(self):
         """
-        :class:`nidaqmx.constants.Impedance1`: Specifies the input
-            impedance of the channel.
+        Impedance1: Specifies the input impedance of the channel.
```

Comment:
Something strange is going on here. Impedance1 isn't a type, but [nidaqmx.constants.Impedance1]https://github.com/ni/nidaqmx-python/blob/master/generated/nidaqmx/constants.py#L383) is. Also the import disappeared at the top of this file, too.